ui/src/views/jobs/mutation/JobMutation.jsx

```diff
@@ -38,7 +38,7 @@ export default function JobMutator() {
   const dispatch = useDispatch();
 
   const isSavingEnabled = () => {
-    return notificationAdapterData.length > 0 && providerData.length > 0 && name != null && name.length > 0;
+    return notificationAdapterData.length && providerData.length && name;
```

Comment:
this is not exactly the same check than I had. in your case, setting name to ' ' would resolve truthy...

Reply:
You're right, it leads to a different result. I didn't notice that, it should be wrapped in a `Boolean` constructor.

The thing is,

```js
console.log(Boolean('')) // false
```

Assuming `notificationAdapter.length=1` and `providerData.length=1` and `name=''`:
```js
console.log(1 && 1 && ''); // ''
```

It works when wrapped though:
```js
console.log(Boolean(1 && 1 && '')); // false
console.log(Boolean(1 && 1 && 'a')); // true
console.log(Boolean(1 && 1 && ''anythingelse)); // true
```